packages/devextreme/js/ui/rating.js

```diff
@@ -0,0 +1,2 @@
+import Rating from '../__internal/ui/m_rating';
+export default Rating;
```

Comment:
Don't forget add //style comment and modify ideal dependency tree here:
https://github.com/Zedwag/DevExtreme/blob/ratingDeclarations__24_2/packages/devextreme-themebuilder/tests/data/dependencies.ts#L2